crmprtd/execution/wmb.sh

```diff
@@ -1,9 +1,17 @@
 #!/bin/bash
 
+# Distinguishes cache and log file names.
+tag="$1"
+
+# Target database DSN, e.g., postgresql://crmprtd@db.pcic.uvic.ca:5433/crmp
+db="$2"
+
 export net=wmb
 export NOW=$(date +'%Y-%m-%dT%H:%M:%S')
 
-export log_file=~/${net}/logs/mof_json.log
-export cache_file=~/${net}/download/${net}_${NOW}.txt
+# TODO: Is `mof` right??? Why not `${net}`?
```

Comment:
"Legacy". This is Ministry of Forests - Wildfire Management Branch. One of the first networks that we incorporated, before we even formalized `crmprtd`. So at the time, there were no naming conventions and that's what it got named. It stuck.